test/base/AppWithDefaultMockFixtures.scala

```diff
@@ -37,4 +42,13 @@ trait AppWithDefaultMockFixtures extends GuiceOneAppPerSuite with BeforeAndAfter
       .overrides(
         bind[IdentifierAction].to[FakeIdentifierAction]
       )
+
+  protected def goodsUnderControlAction(departureIdP5: String,
+                                        mockDepartureP5MessageService: DepartureP5MessageService,
+                                        mockReferenceDataService: ReferenceDataService
+  ): Unit =
+    when(mockGoodsUnderControlActionProvider.apply(any())) thenReturn new FakeGoodsUnderControlAction(departureIdP5,
```

Comment:
Could be possibly do this at the controller level? Keep in mind that this AppWithDefaultFixtures is also used on the P4 pages. Makes sense to keep it locally to the controller we are using rather than globally.